VideoAPI/Testing.Common/Helper/ZAP.cs

```diff
@@ -0,0 +1,168 @@
+using Microsoft.Extensions.Configuration;
+using OWASPZAPDotNetAPI;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Threading;
+using Testing.Common.Configuration;
+
+namespace Testing.Common.Helper
+{
+    public static class ZAP
+    {        
+        private static readonly ClientApi _api = new ClientApi(ZapConfiguration.ApiAddress, ZapConfiguration.ApiPort, ZapConfiguration.Apikey);
+        private static IApiResponse _apiResponse;
+
+        private static ZAPConfiguration ZapConfiguration
```

Comment:
GetZapConfiguration ?

Reply:
Its not a function and just being a property named it ZapConfiguration